docs/dev/guides/me-compiler-pass.md

```diff
@@ -0,0 +1,46 @@
+---
+title: "Add Custom Compiler Passes (ME)"
+description: "How to add a custom compiler pass in the Contao Managed Edition."
+aliases:
+  - /guides/me-compiler-pass.md
```

Comment:
Alias should be
```
/guides/me-compiler-pass/
```
Also I think the filename should be `add-compiler-pass`. Yes, it is about the managed addition, but I would still like that filename/slug better ;)

Reply:
I changed the name/content of the guide to be more general - could you check again if you like the naming and if I did the right thing with the alias?  :slightly_smiling_face: